lti_emailer/canvas_api_client.py

```diff
@@ -107,7 +131,3 @@ def _get_authorized_teaching_staff_enrollment_types(account_id):
             role_name, account_id, canvas_api_helper_courses.COURSE_PERMISSION_SEND_MESSAGES_ALL
         )
     ]
-
-
-def _copy_user_attributes_to_enrollment(user, enrollment):
-    enrollment.update({a: user[a] for a in USER_ATTRIBUTES_TO_COPY})
```

Comment:
Yup.  I couldn't figure out if there was an ordering to the methods in this module, so the first thing I did (as its own commit, https://github.com/Harvard-University-iCommons/lti_emailer/commit/f48de64b5e0ed41be5cfaf0b11fbf3b738901a82, so we could hopefully isolate it in PRs) was to sort the methods alphabetically.  The functional change is the addition of `get_courses_for_account_in_term`.